src/abstractdatatable/show.jl

```diff
@@ -64,10 +64,6 @@ end
 ourshowcompact(io::IO, x::Any) = showcompact(io, x) # -> Void
 ourshowcompact(io::IO, x::AbstractString) = print(io, x) # -> Void
 ourshowcompact(io::IO, x::Symbol) = print(io, x) # -> Void
-ourshowcompact{T<:AbstractString}(io::IO, x::CategoricalValue{T}) =
```

Comment:
See if this deletion is why categorical values are printing strange?

Reply:
Ah, right. Ideally this method wouldn't be needed, but we can keep it if it works around the bug on Julia 0.7.